students/yujeong/users/views.py

```diff
@@ -49,15 +51,20 @@ def post(self, request):
         data = json.loads(request.body)
 
         try: 
-            if not User.objects.filter(email=data['email']).exists() :
-                return JsonResponse({"message": "INVALID_USER"}, status = 401)   
-
-            if not User.objects.filter(email=data['email'], password=data['password']):
-                    
+            email    = data['email']
+            password = data['password'] 
 
-                return JsonResponse({"message": "INVALID_USER"}, status = 401)
+            user = User.objects.get(email=email)
+            
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
+               return JsonResponse({"message": "INVALID_USER_Password error"}, status = 401)    
```

Comment:
로그인 에러 같은 경우 아이디와 비밀번호 중에 어떤 것이 잘 못 되었는지 모르게 하는 것이 좋습니다.
```suggestion
               return JsonResponse({"message": "INVALID_USER"}, status = 401)    
```